hamburger.css

```diff
@@ -0,0 +1,58 @@
+@media (max-width: 768px) {
+  nav .mainMenu {
+    padding: 15px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    color: rgb(255, 255, 255);
+    font-size: 32px;
+    list-style: none;
+    height: 100vh;
+    line-height: 60px;
+    position: fixed;
+    padding-top: 50px;
+    right: 1px;
+    left: 1px;
+    z-index: 10;
+    flex-direction: column;
+    padding-left: 35px;
+    background: #6070ff;
+    opacity: 0.9;
+    transition: top 1s ease 0s;
+    display: none;
+  }
```

Comment:
- [ ] Your `font-weight` still does not match with the Figma template. Kindly change that as the previous Code Reviewer pointed out.